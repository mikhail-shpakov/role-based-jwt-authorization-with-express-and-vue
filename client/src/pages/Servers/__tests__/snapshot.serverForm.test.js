import { createLocalVue, shallowMount } from '@vue/test-utils'
import Buefy from 'buefy'
import serverForm from '../serverForm'

describe('./ServerForm.vue', () => {
  const localVue = createLocalVue()
  localVue.use(Buefy)

  const selected = {
    user_id: 'user5',
    server_name: 'server2',
    server_type: 'dedicated'
  }

  it('when load page, form renders correctly', () => {
    const wrapper = shallowMount(serverForm, {
      localVue,
      propsData: { selected }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
