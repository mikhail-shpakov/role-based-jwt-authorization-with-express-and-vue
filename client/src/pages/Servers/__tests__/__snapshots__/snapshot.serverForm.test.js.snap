// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./serverForm.vue when load page, form renders correctly 1`] = `
<div class="container">
  <div class="title is-size-4">Редактировать</div>
  <form class="box">
    <p><strong>ID:</strong> user5</p>
    <div class="divider"></div>
    <b-field-stub label="Название сервера" addons="true">
      <b-input-stub expanded="true" usehtml5validation="true" value="server2" type="text" hascounter="true" customclass="" required="required"></b-input-stub>
    </b-field-stub>
    <b-field-stub label="Тип сервера" addons="true" class="input-margin">
      <b-select-stub expanded="true" usehtml5validation="true" value="dedicated">
        <option value="dedicated">dedicated</option>
        <option value="vds">vds</option>
        <option value="hosting">hosting</option>
      </b-select-stub>
    </b-field-stub>
    <b-button-stub type="is-success" nativetype="button" tag="button" disabled="true" class="is-pulled-right button-custom">Сохранить изменения</b-button-stub>
  </form>
</div>
`;
